generators/package/index.ts

```diff
@@ -61,12 +61,12 @@ module.exports = class extends HtmlGenerator {
     }
     
     if (fs.existsSync(this.destinationPath(webAppFolder, "node_modules"))) {
-      const excludeFilter = (folder) => { return folder.startsWith("cmf") && ["cmf.taura", "cmf.core", "cmf.core.multicast.client", "cmf.mes", "cmf.polyfill", "cmf.angular", "cmf.instascan", "cmf.core.examples", "cmf.mes.examples"].indexOf(folder) < 0 && !folder.startsWith("cmf.style") };
+      const excludeFilter = (folder) => { return folder.startsWith("cmf") && ["cmf.taura", "cmf.core", "cmf.core.multicast.client", "cmf.mes", "cmf.polyfill", "cmf.angular", "cmf.instascan", "cmf.core.examples", "cmf.mes.examples"].indexOf(folder) < 0 && !folder.startsWith("cmf.style") || folder === "@angular"; };
```

Comment:
although boolean logic works, please wrap around parenthesis to make it more readable (same below)